src/actions/onboardingActions.js

```diff
@@ -203,15 +214,18 @@ const finishRegistration = async ({
   });
 };
 
-const navigateToAppFlow = (isWalletBackedUp: boolean, accountId: string) => {
+const navigateToAppFlow = (isWalletBackedUp: boolean, getState: GetState) => {
+  const accounts = getState().accounts.data;
+  const keyBasedAccount = findKeyBasedAccount(accounts);
+  const accountId = keyBasedAccount ? getAccountId(keyBasedAccount) : '';
```

Comment:
Maybe `keyBasedAccount && getAccountId(keyBasedAccount)`? Because `getAccountId` will already return empty string if not found.

Reply:
getAccountId() requires an account to be passed, while findKeyBasedAccount() could return null